### PR TITLE
Implement Auto Color Theme

### PR DESCRIPTION
**Pull request by nikischin**
PR for FR #48

### CODE REVIEW

File: src/util/parameters.ts [hunk 1]
```diff
@@ -2,20 +2,24 @@
  * Color schemes of the map.
  * @see {@link https://developer.apple.com/documentation/mapkitjs/map/colorschemes}
  */
-export enum ColorScheme { Light, Dark }
+export enum ColorScheme { Light, Dark, Auto }
 
 /**
  * Converts a mapkit-react color scheme value to a MapKit JS color scheme value.
  * Must be called after MapKit JS is loaded.
  * @param colorScheme The mapkit-react color scheme value
```
Comment (Nicolapps) [inline src/util/parameters.ts:10]: Could you please document the new parameter here?

File: src/components/MapProps.tsx [hunk 2]
```diff
@@ -18,6 +18,8 @@ export default interface MapProps {
 
   /**
    * The map’s color scheme when displaying standard or muted standard map types.
+   * Offers the Options ColorScheme.Dark, ColorScheme.Light and ColorScheme.Auto
+   * Auto Color Theme does decide on the colorscheme for mapkit based on the browser preference
```
Comment (Nicolapps) [inline src/components/MapProps.tsx:22]: A small suggestion about grammar/wording:

```suggestion
   * Offers the options ColorScheme.Dark, ColorScheme.Light and ColorScheme.Auto.
   * Auto will select automatically the appropriate MapKit color scheme based on the browser preference.
```

**Review by Nicolapps — COMMENTED** [review]
Thank you for the PR! I have a couple suggestions about documentation, we can merge the PR after this.

**Comment by nikischin** [comment]
> Thank you for the PR! I have a couple suggestions about documentation, we can merge the PR after this.

Thank you so much for your improvements! As I am not native english Speaking, I am very happy about any improvement in wording!

**Review by Nicolapps — APPROVED** [review]
Thanks for the PR!

**Pull request merged**